Assets/Scripts/Controllers/BuildModeController.cs

```diff
@@ -190,5 +193,21 @@ public void DoBuild(Tile t)
 
     }
 
+    public bool BuildJobWouldOverlapExistingBuildJob(Tile t, string furnitureType)
```

Comment:
maybe change function name to something like "DoesBuildJobOverlapExistingBuildJob" since it makes it more clear that it returns a bool by making the name a question.